wiki/Help_Centre/Client/en.md

```diff
@@ -138,9 +138,11 @@ To safely force your beatmap database to be recreated, try the following steps:
 4. Close osu!.
 5. Find the `osu!.db` file located in the folder you just opened.
 6. Right-click on it, then click `Rename`.
-7. Rename it to anything. It does not matter what you rename it to, as long as it is not "osu!". Then press `Enter`.
+7. Rename it as you wish. It doesn't matter, as long as it is not called "osu!". Then press `Enter`.
 8. Relaunch osu!.
 
+*Note: since version [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19, this problem seems to become less frequent.*
```

Comment:
Has it actually been confirmed to still be happening ever? We put a lot of effort into making sure it was fixed once-and-for-all, so may be worth removing this completely if that's the case?

Reply:
In any case for me this problem has not appeared since the last version, because I was downloading a lot of beatmaps at the same time, and I often had this status problem, and since this update, it has never come back.

After that I hesitated to delete this section completely, because if the problem comes back, there is no solution in the Help center, which would be annoying. 
But what might be possible would be to keep it in the article but put it between `<!-- -->` like what has already been done in `Help_centre/Website`, which seems to me to be a good compromise; not to totally delete it, and not to keep it either.

Reply:
Let's leave it for the time being, I guess. How about changing this note to the following:

```suggestion
*Note: a fix for this was added in [Stable 20210519.3](https://osu.ppy.sh/home/changelog/stable40/20210519.3) dated 2021-05-19. If you happen to still encounter this issue, please let us know at https://github.com/ppy/osu-stable-issues/issues.*
```

Reply:
I agree.